canonical-web-and-design/certification.ubuntu.com
# Wrapper templates

# Done
Creates the following components:
`header.html`
`footer.html`
`layout.html`
to wrap the content.

Fixes https://github.com/canonical-web-and-design/base-squad/issues/620

# QA
`./run` and make sure that the header and footer look exactly the same as on https://certification.ubuntu.com
